### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Settings_Disk_Settings_Transfers_ScreenReader: When the voiceover tab focus lands on radio buttons  'Adjust Dynamically', 'Based on number of CPU cores' and 'Limit to' voiceover announces them all as  '1 of 1' which is incorrect as there are 3 listed items in one list.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/AzureStorageExplorer#5779]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_AzureToolsStorageExplorer_May2022;#Azure Tools Storage Explorer;#MAC;#FTP;#DesktopApp;#WCAG1.3.1;#Voiceover;#AINotInScope;

## Environment Details:
Application Name: Azure Tools Storage Explorer
Application Version: 1.24.0
Mac Version: macOS Monterey 12.3.1
Screen Reader: Voiceover

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch Azure Tools Storage Explorer.
3. Sign in to Azure.
4. Tab to 'Settings' pane tree view.
5. Tab to 'Transfers'
6. Tab till 'Adjust Dynamically' radio button under 'Network Concurrency'
7. Turn on voiceover and observe.

## Actual:
When the voiceover tab focus lands on radio buttons 'Adjust Dynamically', 'Based on number of CPU cores' and 'Limit to' voiceover announces them all as '1 of 1' which is incorrect as there are 3 listed items in one list.

## Expected:
When the voiceover focus lands on 'Adjust Dynamically', 'Based on number of CPU cores', 'Limit to' radio button, voiceover should announce the listed number for each of them as '1 of 3, 2 of 3 and 3 of 3' as there are 3 radio buttons under one category.

## User Impact:
Screen reader users will face difficulty in perceiving user interface component.

## Attachment:
![Presentation1](https://user-images.githubusercontent.com/98393966/170236022-8d8d569b-72cb-44ae-97b6-3066003d3055.gif)
